src/components/soknadpanel/SoknadPanel.tsx

```diff
@@ -31,70 +28,34 @@ function SoknadPanel({ soknad }: Props): JSX.Element {
-                                data?.sykmelding?.arbeidsgiver.navn
-                            } i løpet av perioden ${formatDateRange(soknad.fom, soknad.tom)}?`}
-                            value={false}
-                        />
-                        <ListItemCheckbox
-                            title={`Hadde du ferie i tidsrommet ${formatDateRange(soknad.fom, soknad.tom)}?`}
-                            value={soknad.fravar?.some((it) => it.type === 'FERIE')}
-                        />
-                        <ListItemCheckbox
-                            title={`Tok du permisjon mens du var sykmeldt ${formatDateRange(soknad.fom, soknad.tom)}?`}
-                            value={soknad.fravar?.some((it) => it.type === 'PERMISJON')}
-                        />
-                        <ListItemCheckbox
-                            title={`Var du på reise utenfor EØS mens du var sykmeldt ${formatDateRange(
-                                soknad.fom,
-                                soknad.tom,
-                            )}?`}
-                            value={soknad.fravar?.some((it) => it.type === 'UTLANDSOPPHOLD')}
-                        />
-                        <ListItemCheckbox
-                            title={`I perioden ${formatDateRange(
-                                soknad.fom,
-                                soknad.tom,
-                            )} var du {TODO: GRAD} sykmeldt fra
-                        ${data?.sykmelding?.arbeidsgiver.navn}. Jobbet du noe i denne perioden? `}
-                            value={false}
-                        />
-                        <ListItemCheckbox
-                            title={`Har du vært under utdanning i løpet av perioden ${formatDateRange(
-                                soknad.fom,
-                                soknad.tom,
-                            )}?`}
-                            value={soknad.fravar?.some(
-                                (it) => it.type === 'UTDANNING_DELTID' || it.type === 'UTDANNING_FULLTID',
-                            )}
-                        />
+                        {soknad.sporsmal
+                            .filter((sporsmal) => {
+                                return skalSporsmalVises(sporsmal);
+                            })
+                            .map((sporsmal, index) => {
+                                return <SporsmalVarianter key={index} sporsmal={sporsmal} />;
+                            })}
                     </ul>
                 </section>
-                <CheckboxExplanation
-                    text="Jeg har lest all informasjonen jeg har fått i søknaden og bekreftet at opplysningene jeg har gitt er korrekte."
-                    alignStart
-                />
             </div>
         </Panel>
     );
 }
 
 export default SoknadPanel;
+
+function skalSporsmalVises(sporsmal: SoknadSporsmalFragment): boolean {
```

Comment:
filtreres disse spørsmålene ut alltids? eller kun her? For om de alltids filtreres ut så kan vi jo filtrere dem ut i resolveren kanskje, så slipper vi å sende over spørsmål vi ikke viser.